data/mods/MindOverMatter/snippets/snippets_new.json

```diff
@@ -278,7 +357,7 @@
       {
         "name": "To all employees: re: GASPAR",
         "id": "lab_file_phavian_memo_01",
-        "text": "Colleagues,\n\nI'd like to take a moment to address the recent rumors.  I know that stress levels are high with the quarantine and the XE037 news, but there is no need to engage in unwarranted speculation.  In these unprecedented times, it's important to remember that we're all in this together and we will solve the problems before us, as we have no many others.\n\nI will be perfectly clear.  There is no \"Project GASPAR.\"  Trained mathematicians are the most important resource that we have, and it would be the height of folly to waste them on replicating a resource that already exists simply because the current state of emergency has placed us out of contact.  Furthemore, it is important to remember that Netherum mathematics are not magic.  They obey a consistent set of rules and we are capable of determining those rules empirically.  We can test how much an ONYX LIGHTNING mathematician can lift to the milligram.  We can determine the effective communication range for a GRAY DAWN candidate down to the decimeter.  We would never simply \"throw things at the wall and see what sticks.\"  Science requires rigor.\n\nThese are troubling times, but we will get through them together.\n\nSincerely,\nHarold Liu\nDirector, Project PHAVIAN"
+        "text": "Colleagues,\n\nI'd like to take a moment to address the recent rumors.  I know that stress levels are high with the quarantine and the XE037 news, but there is no need to engage in unwarranted speculation.  In these unprecedented times, it's important to remember that we're all in this together and we will solve the problems before us, as we have no many others.\n\nI will be perfectly clear.  There is no \"Project GASPAR.\"  Trained mathematicians are the most important resource that we have, and it would be the height of folly to waste them on replicating a resource that already exists simply because the current state of emergency has placed us out of contact.  Furthemore, it is important to remember that Netherum mathematics are not magic.  They obey a consistent set of rules and we are capable of determining those rules empirically.  We can test how much a JUNIPER GOLD mathematician can lift to the milligram.  We can determine the effective communication range for a GRAY DAWN candidate down to the decimeter.  We would never simply \"throw things at the wall and see what sticks.\"  Science requires rigor.\n\nThese are troubling times, but we will get through them together.\n\nSincerely,\nHarold Liu\nDirector, Project PHAVIAN"
```

Comment:
```suggestion
        "text": "Colleagues,\n\nI'd like to take a moment to address the recent rumors.  I know that stress levels are high with the quarantine and the XE037 news, but there is no need to engage in unwarranted speculation.  In these unprecedented times, it's important to remember that we're all in this together and we will solve the problems before us, as we have so many others.\n\nI will be perfectly clear.  There is no \"Project GASPAR.\"  Trained mathematicians are the most important resource that we have, and it would be the height of folly to waste them on replicating a resource that already exists simply because the current state of emergency has placed us out of contact.  Furthermore, it is important to remember that Netherum mathematics are not magic.  They obey a consistent set of rules and we are capable of determining those rules empirically.  We can test how much a JUNIPER GOLD mathematician can lift to the milligram.  We can determine the effective communication range for a GRAY DAWN candidate down to the decimeter.  We would never simply \"throw things at the wall and see what sticks.\"  Science requires rigor.\n\nThese are troubling times, but we will get through them together.\n\nSincerely,\nHarold Liu\nDirector, Project PHAVIAN"
```

Reply:
no many others to so many others and furthermore was mispelled since the gitdif isn't showing the changes.